utils/check-token.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+function checkToken(token) {
+
+  if (!token || !token.match(/^(sk|pk)\.([^/]*)$/)) {
```

Comment:
should `tk` be included?

Reply:
Not sure home frequently a `tk` token is needed for this use case, but can definitely add one 👍

Reply:
added in c8d4937